sci_analysis/data/numeric.py

```diff
@@ -142,6 +142,7 @@ def drop_nan(self):
         arr : pandas.DataFrame
             A copy of the Numeric object's internal Series with all NaN values removed.
         """
+        self._dropped_vals = self._values[self._dep].isnull()
```

Comment:
Should this be `self._values[self._ind].isnull()`? In drop_nan(), the dependent column is probably nan for each value.